__tests__/app.test.js

```diff
@@ -331,20 +331,45 @@ describe("PATCH /api/articles/:article_id", () => {
         expect(body.msg).toBe("Bad Request");
       });
   });
-   test("404: Responds with Not Found if the article_id does not exist", () => {
-     return request(app)
-       .patch("/api/articles/2000")
-       .expect(404)
-       .then((res) => {
-         expect(res.body.msg).toEqual("Not Found");
-       });
-   });
-   test("400: Responds with Bad Request when passed an article_id that is not a number", () => {
-     return request(app)
-       .patch("/api/articles/notANumber")
-       .expect(400)
-       .then((res) => {
-         expect(res.body.msg).toBe("Bad Request");
-       });
-   });
+  test("404: Responds with Not Found if the article_id does not exist", () => {
+    return request(app)
+      .patch("/api/articles/2000")
+      .expect(404)
+      .then((res) => {
+        expect(res.body.msg).toEqual("Not Found");
+      });
+  });
+  test("400: Responds with Bad Request when passed an article_id that is not a number", () => {
+    return request(app)
+      .patch("/api/articles/notANumber")
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe("Bad Request");
+      });
+  });
+});
+
+describe.only("DELETE /api/comments/:comment_id", () => {
+  test("200: Deletes the comment of the specified comment_id", () => {
+    return request(app)
+      .delete("/api/comments/1")
+      .expect(204)
+      .then(() => {
+        return request(app)
+          .get("/api/articles/9/comments")
+          .expect(200)
+          .then(({ body }) => {
+            console.log("I'm in the get statememt");
```

Comment:
Great to see that you are using console logs to help you write your code, just try to remember to clean them up before pushing your work.